package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "chromium-pickle-js": "^0.2.0",
-    "commander": "^4.1.1",
+    "commander": "^5.0.0",
     "glob": "^7.1.6",
     "minimatch": "^3.0.4"
```

Comment:
const { app, globalShortcut } = require('electron')

app.whenReady().then(() => {
  // Register a 'CommandOrControl+Y' shortcut listener.
  globalShortcut.register('CommandOrControl+Y', () => {
    // Do stuff when Y and either Command/Control is pressed.
  })
})

Reply:
Super